File: demo/app/app/src/main/java/com/vectordemo/domain/model/oss/OssFileContentUpdateModel.kt
package com.vectordemo.domain.model.oss

import com.vectordemo.utils.json.GsonBean

data class OssFileContentUpdateModel(
    val fileId: String,
    val originFileName: String,
    val url: String,
    val updated: Boolean,
    val message: String
) : GsonBean

File: demo/flutter/flutternew/lib/domain/model/oss/oss_file_content_update_model.dart
class OssFileContentUpdateModel {
  final String fileId;
  final String originFileName;
  final String url;
  final bool updated;
  final String message;

  const OssFileContentUpdateModel({
    required this.fileId,
    required this.originFileName,
    required this.url,
    required this.updated,
    required this.message,
  });
}
